static/app/views/eventsV2/metricsBaselineContainer.tsx

```diff
@@ -198,7 +197,7 @@ export function MetricsBaselineContainer({
                 lineStyle: {
                   color: additionalSeriesColor[order],
                   type: 'dashed',
-                  width: 1,
+                  width: 1.5,
```

Comment:
I also increased the line width to make it more visible @olivier-w just wanted to run that change by you too!

Reply:
Looks great to me!